src/lib/layer-manager.js

```diff
@@ -594,6 +594,10 @@ export default class LayerManager {
    *                        {Object} srcEvent:             native JS Event object
    */
   _onPointerMove(event) {
+    if (event.isDown) {
+      // Is dragging
```

Comment:
Well, more specifically `_onPointerMove` feeds into our hover handlers (`onHover` and `onLayerHover`), and dragging is not supported by the framework itself, it's the app's responsibility. But yeah, fleshing out the comment will help future detective work.